yt/yt/client/table_client/helpers.cpp

```diff
@@ -1568,4 +1569,49 @@ REGISTER_INTERMEDIATE_PROTO_INTEROP_BYTES_FIELD_REPRESENTATION(
 
 ////////////////////////////////////////////////////////////////////////////////
 
+TUnversionedValueRangeTruncationResult TruncateUnversionedValues(TUnversionedValueRange values, i64 size)
+{
+    auto rowBuffer = New<TRowBuffer>();
```

Comment:
Do we want to allocate a row buffer for each sample? It might end up with many small allocations.